doc/tutorials/distributed.rst

```diff
@@ -91,6 +93,17 @@ If you are using Azure Confidential Computing, all your VMs must be on the same
       mc2-xgboost/host/dmlc-core/tracker/dmlc-submit --cluster ssh --host-file hosts.config --num-workers <num_nodes_in_cluster> --worker-memory 4g python3 distr-training.py
 
 
+   The command takes in the following arguments:
+
+         * ``--cluster`` : how the cluster is set up. In Secure XGBoost we leverage SSH for intra-cluster communication. 
```

Comment:
Do we wanna list the possible values for this argument? e.g. `ssh` vs `local`. 
Or better still, is there a help command that dumps all this info?

Reply:
We could list it. There should be a help command, but all the possible arguments are also listed here: https://github.com/dmlc/dmlc-core/blob/master/tracker/dmlc_tracker/opts.py. The 4 I listed are really all we need though, so I'm not sure if it'd be helpful to list all possible arguments and their possible values.

I'll add a note about a local cluster though.